python/trigger_ghebackup.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import subprocess
+import sys
+
+
+def get_pid(name: str) -> list:
+    """Returns a list of pids, can be empty.
+    @attention: inspired by http://stackoverflow.com/a/26688998 on 09/02/2017
+    @author: Padraic Cunningham"""
+    if name:
+        return list(map(int, subprocess.check_output(["pidof", name]).split()))
+    return []
+
+
+def start_ghe_backup(name: str) -> None:
+    """
+    Spawns process specified by given name
+    :param name: process to be spawned
+    """
+    try:
+        subprocess.Popen([name, '-v', '1>>', '/var/log/ghe-prod-backup.log', '2>&1'])
```

Comment:
You probably want to run this with `shell = True` and provide a string instead of a list, otherwise the output will not work.

You can also use [subprocess.run](https://docs.python.org/3/library/subprocess.html#subprocess.run):

```python
process = subprocess.run([name, '-v'], stdout=subprocess.PIPE,  stderr=subprocess.STDOUT)

with open('/var/log/ghe-prod-backup.log', 'a+') as log_file:
    log_file.write(process.output)
```

The second approach will block until the backup process is finished.